modules/programs/vscode.nix

```diff
@@ -187,6 +189,34 @@ in {
           or by Visual Studio Code.
         '';
       };
+
+      userSnippets = mkOption {
```

Comment:
Should probably be named `languageSnippets`?

Reply:
I've renamed the option to ```languageSnippets```.